suite-native/state/src/extraDependencies.ts

```diff
@@ -25,6 +25,8 @@ const transports = transportsPerDeviceType[deviceType];
 
 export const extraDependencies: ExtraDependencies = mergeDeepObject(extraDependenciesMock, {
     selectors: {
+        // using all enabled networks even those current device does not support,
```

Comment:
why is this comment here? Should not it be rather at the place where is this extra dep used?

Reply:
It feels obsolete to have it in disableAccountsThunk, because that is the place where you actually expect this behaviour. I've added the info here to make sure that we know why this selector was used over the device specific one.